db/migrate/20210530122323_move_triggers_to_fields.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class MoveTriggersToFields < ActiveRecord::Migration[6.1]
+  def change
+    add_column :discourse_automation_automations, :trigger, :string, null: true
+
+    add_column :discourse_automation_fields, :target, :string, null: true
+
+    DB.exec(
+      <<~SQL
+        UPDATE discourse_automation_fields
+        SET target = 'script'
```

Comment:
Is that the default value? If it is, then maybe we could set it as the default value in the column definition instead?

Reply:
no it's not default, it's just all the existing where scripts